packages/itwinui-react/src/core/FileUpload/FileUploadTemplate.tsx

```diff
@@ -51,11 +68,52 @@ export const FileUploadTemplate = (props: FileUploadTemplateProps) => {
     acceptType,
     label = 'Choose a file',
     subtitle = 'or drag & drop it here.',
+    files,
+    fileLabel,
+    fileDescription,
+    svgIcon,
     children,
   } = props;
   useTheme();
 
-  return (
+  const icon = svgIcon ? (
+    React.cloneElement(svgIcon, {
+      className: cx('iui-icon', svgIcon.props.className),
+      'aria-hidden': true,
+      'data-iui-file-uploaded': true,
+    })
+  ) : (
+    <SvgDocument
+      className='iui-icon'
+      aria-hidden
+      data-iui-file-uploaded={true}
+    />
+  );
+
+  return files?.length === 1 ? (
+    <>
+      {icon}
```

Comment:
This has been exported into a separate component. I also have the option for users to use the `FileUploaded` component without needing the `FileUpload` or `FileUploadTemplate` components, but a few CSS props are not applied to `FileUploaded` when its not nested within `FileUpload` since they are applied to `iui-content`

```SCSS
    padding: calc(var(--iui-size-s) * 0.5) var(--iui-size-s);
    border: 1px solid var(--iui-color-background-backdrop);
```

Also the additional props for `iui-drag` would not be applied to `FileUploaded` when used as a standalone component